lib/hammer_cli_foreman/combination.rb

```diff
@@ -45,8 +45,8 @@ class ListCombination < HammerCLIForeman::ListCommand
       end
 
       build_options do |o|
-        o.expand(:all).except(:config_templates, :hostgroups, :environments)
-        o.without(:config_template_id, :hostgroup_id, :environment_id)
```

Comment:
is the removal from the `o.without` here and below intentional?

Reply:
@tbrisker, yes @xprazak2 mentioned that the template_id should be in the command.